src/components/tasks/TaskList.jsx

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from "react-redux";
+import TaskItem from "src/components/taskItem/TaskItem";
+import "src/components/tasks/TaskList.component.scss";
+
+const TaskList = () => {
+  const tasks = useSelector((state) => state.todos);
+
+  return (
+    <ul className="task-list">
+      {tasks.map((task) => {
+        return <TaskItem key={task.id} task={task} />;
+      })}
+    </ul>
```

Comment:
noticed that you have used unordered list. But the component is card item instead of list item. Can we change this to according?